include/sys/signal.h

```diff
@@ -143,6 +143,11 @@ int sig_send(signo_t sig, sigset_t *mask, sigaction_t *sa, ksiginfo_t *ksi);
  * \note This is machine dependent code! */
 int sig_return(void);
 
+/*! \brief Perform signal-related tasks on process exec.
```

Comment:
I think `sig_onexec` name is already self-descriptive and there's no need to repeat it here. Perhaps you could write what this procedure actually does (i.e. resets handlers).